m4_sessions/1710-inning/group_one.md

```diff
@@ -1,10 +1,27 @@
-Group One
+# Group One
+## Session Title: Toxic Masculinity & Nerd Culture
+## Description: Why should students come to this session?
+* Nerd culture is now part of the zeitgeist. Come join us and 		learn why you got to get over your idea of what a nerd is, 		dude.
 
-Session Title: 
-Description: Why should students come to this session? 
+## Session Outcomes: What will students leave with or gain from this session?
+* Intro: What’s your favorite nerd thing?
+* Awareness of what nerd culture encompasses and why the term ‘nerd’ isn’t specific.
+* Awareness of how toxic masculinity effects media and why there’s this protectionism in ‘nerd’ media (aka. misogyny)
+* Examples in media of this actually in media
+* Real world examples of ways it can harm (Dr. Disrespect)
+* Discussion - Where do you see the effects in media and in your own life? How has this affected you? 
+* Awareness of why you are now ONE OF US
 
-Session Outcomes: What will students leave with or gain from this session?
+## Session Outline: Include the flow of the session and the time for each section.
+* Intro - Introduce selves with favorite piece of nerd media/thing
+* The evolution of nerd culture
+* The protectionism of the term ‘nerd’ and why it isn’t specific - point to how we all like different things considered nerdy and toxic masculinity in these things
+* How this has influenced media and our own lives 
+* Small groups with questions
+* Bring it back together and reflections 
```

Comment:
Final reflections:  How can you bring the best of nerd culture?  How can you be a part of changing the toxic masculinity?